package/src/components/AccountProfileInfo/v1/AccountProfileInfo.js

```diff
@@ -1,40 +1,148 @@
 import React, { Component } from "react";
 import PropTypes from "prop-types";
 import styled from "styled-components";
-import { applyTheme } from "../../../utils";
+import { addTypographyStyles, applyTheme } from "../../../utils";
 
-const StyledDiv = styled.div`
-  color: ${applyTheme("accountProfileInfoColor")};
+const AccountProfileInfoContainer = styled.div`
+  display: flex;
+  position: relative;
 `;
 
+const AccountProfileInfoTextContainer = styled.div`
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  position: relative;
+`;
+
+const ViewerInitialsCircle = styled.div`
+  background-color: ${applyTheme("viewerInfoInitialsBackgroundColor")};
+  border-radius: 50%;
+  height: ${applyTheme("accountProfileInfoInitialsSize")};
+  text-align: center;
+  width: ${applyTheme("accountProfileInfoInitialsSize")};
+`;
+
+const ViewerInitialsText = styled.div`
+  ${addTypographyStyles("ViewerInfoInitials", "labelText")}
+  color: ${applyTheme("viewerInfoInitialsColor")};
+  line-height: 1;
+  position: relative;
+  top: calc(${applyTheme("viewerInfoInitialsSize")} / 4);
+`;
+
+const ViewerEmailText = styled.span`
+  ${addTypographyStyles("ViewerInfoInitials", "labelText")}
+  color: ${applyTheme("accountProfileInfoEmailFontColor")};
+  font-size: ${applyTheme("accountProfileInfoEmailFontSize")};
+  align-self: left;
+  margin-bottom: 0.75rem;
+  margin-left: 1.0rem;
```

Comment:
Per the new guidelines, we should almost never be using `rem`. Use `em` if it should grow/shrink with the text size in this element, or `px` if it should remain the same as the text size in this element.